CRM/Contact/Form/Task.php

```diff
@@ -86,7 +86,7 @@ public static function preProcessCommon(&$form) {
     $form->_contactIds = [];
     $form->_contactTypes = [];
 
-    $isStandAlone = in_array('task', $form->urlPath) || in_array('standalone', $form->urlPath);
+    $isStandAlone = in_array('task', $form->urlPath) || in_array('standalone', $form->urlPath) || in_array('map', $form->urlPath);
```

Comment:
I think we could call $form->isStandAlone() & then on the CRM_Contact_Form_Task_Map form it could return TRUE

Reply:
@eileenmcnaughton is that a new function or just something I haven't come across but yeh I'm thinking that might be what is needed, the map task does get used in a number of different contexts so will need to check if this does anything to them (don't think so but maybe)